plugin/parser/engine/pg/convert.go

```diff
@@ -902,6 +908,28 @@ func convertToDecimal(typmods []*pgquery.Node) ast.DataType {
 	}
 }
 
+func convertToVarchar(typmods []*pgquery.Node) ast.DataType {
+	if len(typmods) != 1 {
+		return &ast.UnconvertedDataType{}
```

Comment:
Just realized we may add a comment field to UnconvertedDataType to describe the reason why it's not converted.

e.g. in this block, it's unclear why we can't convert len(typmods) !=1

Reply:
I'll add a field to describe it in the next PR.